modules/va_forms/app/sidekiq/va_forms/form_builder.rb

```diff
@@ -7,37 +7,59 @@ module VAForms
   class FormBuilder
     include Sidekiq::Job
 
-    FORM_FETCH_ERROR_MESSAGE = 'The form could not be fetched from the url provided.'
+    class FormFetchError < StandardError; end
+
     STATSD_KEY_PREFIX = 'api.va_forms.form_builder'
 
     sidekiq_options retry: 7
 
-    sidekiq_retries_exhausted do |msg, error|
-      args = msg['args'] || {}
-      form_name = args['fieldVaFormNumber']
-      row_id = args['fieldVaFormRowId']
+    sidekiq_retries_exhausted do |msg, _ex|
+      job_id = msg['jid']
+      error_class = msg['error_class']
+      error_message = msg['error_message']
+
+      form_data = msg['args'].first
+      form_name = form_data['fieldVaFormNumber']
+      row_id = form_data['fieldVaFormRowId']
 
       # Ensure that the form is marked "valid_pdf: false" if successive form fetches have failed
-      if error.message == FORM_FETCH_ERROR_MESSAGE
+      if error_class.to_s == FormFetchError.to_s
         form = VAForms::Form.find_by(row_id:)
-        url = VAForms::Form.normalized_form_url(args.dig('fieldVaFormUrl', 'uri'))
+        url = VAForms::Form.normalized_form_url(form_data.dig('fieldVaFormUrl', 'uri'))
         form_previously_valid = form.valid_pdf
         form.update!(valid_pdf: false, sha256: nil, url:)
         if form_previously_valid
           VAForms::Slack::Messenger.new(
             {
-              class: self.class.name,
+              class: 'VAForms::FormBuilder',
```

Comment:
I had noticed that no other jobs that are using a `sidekiq_retries_exhausted` block refer to `self` inside the block. I have some concern that it doesn't behave as it would elsewhere, so I have made this change just to be safe.